packaging/opensuse/snapd.spec

```diff
@@ -393,6 +393,8 @@ fi
 %{_libexecdir}/snapd/snap-device-helper
 %{_libexecdir}/snapd/snap-discard-ns
 %{_libexecdir}/snapd/snap-exec
+# Not making snap-verify available as its specialized for Ubuntu Core systems
```

Comment:
This is incorrect because it is illegal to comment-away %{...} expressions. Just leave this out since both Fedora and openSUSE will fail to build when there are installed files that are not motioned by %files.